kanidmd/src/lib/actors/v1_write.rs

```diff
@@ -140,12 +141,15 @@ impl QueryServerWriteV1 {
                     e
                 })?;
 
+            let f_uuid = filter_all!(f_eq("uuid", PartialValue::new_uuid(target_uuid)));
+            // Add any supplemental conditions we have.
+            let joined_filter = Filter::join_parts_and(f_uuid, filter);
+
             let mdf = match ModifyEvent::from_internal_parts(
                 audit,
                 ident,
-                target_uuid,
```

Comment:
I'm not sure I get this comment? It's removed because in the places we used to specify a target uuid to modify, we now transform that into a pre-built filter which makes this more flexible.

Reply:
I'm red-green colorblind and I thought this was an addition and not a removal. Oops lol

Reply:
I'm colour blind too :) 

In the review 'files changed', click the gear next to 'jump to' and select split. It puts it left/right instead which may help.